### PR TITLE
Bash script to format all .cpp and .h files

### LINKED ISSUE #99: Add Formatting to the `driverinterface.sh` script
Make sure to use `clang-format` and format all `.cpp` and all `.h` files

### PR DESCRIPTION
**Pull request by VacaTheCow**
Script uses clang-format to format all .cpp and .h files.

### CODE REVIEW

File: driverinterface.sh [hunk 1]
```diff
@@ -1,13 +1,8 @@
 #!/bin/sh 
-cd /home/ucrsolarcar/UCRSolarCar2023/Driver-Interface-2022
 
-res=$(git pull origin main | grep Already)
```
Comment (shahdivyank) [inline driverinterface.sh:4]: You have to add the formatting before the code compiles. The current code is responsible for compiling the code on the raspberry pi and running the application automatically. You will have to add your steps into the current script.

File: driverinterface.sh [hunk 2]
```diff
@@ -1,13 +1,8 @@
...
-if [[ ${res} =~ 'Already' ]]
-then
-  echo 'No updates have been made :)'
-else
-  qmake DriverInterface.pro
-  make
-fi
-
-./DriverInterface
+for FILE in *; do
```
Comment (shahdivyank) [inline driverinterface.sh]: Have you tested this out?

**Review by shahdivyank — CHANGES_REQUESTED** [review]

File: driverinterface.sh (hunk 2 above)
Comment (VacaTheCow) [reply driverinterface.sh]: I did test this out. All .cpp and .h files get formatted. Did you find an error or see that it is incorrect?

File: driverinterface.sh (hunk 1 above)
Comment (VacaTheCow) [reply driverinterface.sh:4]: Apologies for this. I wasn't sure what that code was doing. I have restored the original code and added the clang-format script to it.

**Comment by shahdivyank** [comment]
Move the formatting to after the `git pull` command and right before the `git pull` add a checkout to the production branch

**Review by shahdivyank — APPROVED** [review]

**Review by IseanB — APPROVED** [review]

**Pull request merged**